### DISCUSSION TITLE
eks 설치 후 local kubernetes 가 동작하지 않음

### DISCUSSION THREAD

**Discussion by laigasus** [cloud2scape/.github#35] [on-일반]
## 설명

포팅을 시도하려고 해당 가이드에 따라 선 작업을 수행했었습니다.
https://github.com/cloud2scape/Infra/blob/dev/README.md

다시 로컬 kubespray 로 작업하고자 파일을 돌려놨는데 동작하지 않습니다.

## 오류 발생 재현

1. intellij 에서 
2. kuber-control1 노드의 접속정보를 호스트 디렉터리(`C:\Users\SBA_USER\.kube\config`) 에 두고
3. 호스트에서 `kubectl get nodes` 명령을 수행했을때
4. 동작하지 않습니다.

## 기대 결과

local kubespray 가 다시 동작했으면 좋겠습니다.

##  첨부 자료

```
Starting to serve on 127.0.0.1:52616
E1110 14:26:02.383986   15212 proxy_server.go:147] Error while proxying request: dial tcp 127.0.0.1:6443: connectex: No connection could be made because the target machine actively refused it.
E1110 14:26:03.395002   15212 proxy_server.go:147] Error while proxying request: dial tcp 127.0.0.1:6443: connectex: No connection could be made because the target machine actively refused it.
HTTP GET request to <http://127.0.0.1:52616/apis/authorization.k8s.io/v1/selfsubjectaccessreviews> fails; Server respond: HttpResponse[http://127.0.0.1:52616/apis/authorization.k8s.io/v1/selfsubjectaccessreviews, 500 Internal Server Error]
Proxy with cluster kubernetes-admin@cluster.local was stopped

```

## 나의 환경

**Host**

- OS: windows 11
- IDE: Intellij
- Browser: Chrome
- k8s: v1.31.0

**Vagrant**

- OS: Rockylinux 9.x
- kubespray(control1 + worker3)

**Comment by ssedogdoom** [comment]
로컬에서 kube-apiserver가 제대로 실행되고 있나요?

**Reply by laigasus** [reply]
넵넵.
당장은 kubespray(ubuntu)로 다시 깔고 있습니다...

**Reply by ssedogdoom** [reply]
혹시 버전 문제가 아닐까 하는데요..

참고:https://blog.naver.com/ncloud24/223109710643

**Comment by laigasus** [comment]
127.0.0.1 대신 control node의 ip 주소(ex. 192.168.56.11) 로 수정하여 해결했습니다
eks 설정 후에 config 값이 교체되니까 이점 유의해야겠어요
